front/app/containers/UsersEditPage/DeletionDialog.tsx

```diff
@@ -143,43 +108,27 @@ class DeletionDialog extends PureComponent<Props & InjectedIntlProps, State> {
           <Button
             buttonStyle="delete"
             id="deletion"
-            onClick={this.deleteProfile}
+            onClick={deleteProfile}
             width="auto"
             justifyWrapper="left"
-            processing={processing}
             className="e2e-delete-profile-confirm"
           >
             <FormattedMessage {...messages.deleteMyAccount} />
           </Button>
           <Button
             buttonStyle="text"
-            onClick={this.props.closeDialog}
+            onClick={closeDialog}
             width="auto"
             justifyWrapper="left"
-            processing={processing}
           >
             <FormattedMessage {...messages.cancel} />
           </Button>
         </ButtonsContainer>
```

Comment:
Any specific reason to remove the error message?

Reply:
In the previous component `error` was set to false in the state and it was never changed. So it never rendered.